### PR TITLE
DEP 4.11 dev requirements

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripeltd/product-issues/issues/531

### CODE REVIEW

**Review by GuySartorelli — APPROVED** [review]
LGTM. Merge on green.

File: composer.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
         "php": "^7.4 || ^8.0",
         "silverstripe/recipe-plugin": "^1",
         "silverstripe/recipe-cms": "4.x-dev",
```
Comment (GuySartorelli) [inline composer.json:10]:
```suggestion
        "silverstripe/recipe-cms": "4.11-dev",
```

Comment (emteknetnz) [reply composer.json]: Updated

**Review by GuySartorelli — APPROVED** [review]

**Comment by GuySartorelli** [comment]
Travis still failing on PHP 8.0 and 8.1 - I suspect because silverstripe/silverstripe-travis-shared#40 wasn't merged in yet.

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**